Khan/wonder-blocks
# WB-558 Upgrade to the most recent version of Styleguidist

## Internal
- Upgraded React Storybook to its latest version (9.1.11).
- Updated code examples to match with RS requirements for this new version: Each example needed to import (`require`) all of its dependencies used in each snippet/block.

### Test Plan
1. Open https://deploy-preview-447--wonder-blocks.netlify.com/
2. Check that all the examples are being rendered.

### NOTE:
- Added modules using `require` instead of `import` because this will be done in a separate ticket: WB-559.
- In order to make `import` works correctly, we will need to update `utils/gen-snapshot-tests.js` to change the way we are parsing all the examples.
